test/e2e/container_inspect_test.go

```diff
@@ -46,6 +51,11 @@ var _ = Describe("Podman container inspect", func() {
 		Expect(data).To(HaveLen(1))
 		Expect(data[0].NetworkSettings.Ports).
 			To(Equal(map[string][]define.InspectHostPort{"80/tcp": nil, "8989/tcp": nil}))
+
+		session = podmanTest.Podman([]string{"ps", "--format", "{{.Ports}}"})
```

Comment:
I'm unable to find anywhere that tests `podman ps` on a container with both `-p` and `--expose`. That seems worth doing, but possibly tricky because (IIRC?) sort ordering isn't guaranteed. I've added a TODO item for me to follow up on this, but if you need to repush for any reason, that might be a nice addition here.

Reply:
I can add one, I think the order in the ps output should be stable.